nightly/index.html

```diff
@@ -0,0 +1,215 @@
+        </p>
+    </div>
+
+    <div class="container">
+      <p>
+      <div class="row">
+        <div class="col">
+          <select id="versionSelect" class="form-select required" onchange="onSelectChange()"> <option selected>All Versions</option></select>
+        </div>
+        <div class="col">
+          <select id="platformSelect" class="form-select required" onchange="onSelectChange()"> <option selected>All Platforms</option></select>
+        </div>
+        <div class="col">
+          <select id="architectureSelect" class="form-select required" onchange="onSelectChange()"> <option selected>All Architectures</option></select>
+        </div>
+      </div>
+      </p>
+    </div>
+
+    <div class="container" style="background-color:white;color: black;" id="target"></div>
+    <script type="text/javascript">
+
+      function onSelectChange() {
+        buildPage(jsonText, true);
+      };
+
+      // Compare function for getting order Corretto versions.
+      // Strips off the "Corretto" from CorrettoXX so 8 is 1st.
+      function correttoCompare(a, b) {
+          a_ver=Number(a.slice(8));
+          b_ver=Number(b.slice(8));
+          if(a_ver < b_ver) return -1;
+          if(a_ver > b_ver) return 1;
+          return 0;
+        };
+
+      $(function () {
+            $('#datepicker').datepicker({
+              format: 'yyyy-mm-dd',
+              autoclose: true,
+            }).on('changeDate', function(e) {
+              const corretto_domain = 'https://corretto.aws';
+              const new_url = corretto_domain + '/nightly/' + e.date.getFullYear() + '/' + ("0" + (e.date.getMonth() + 1)).slice(-2) + '/' + ("0" + e.date.getDate()).slice(-2) + '/manifest.json';
+              $(this).datepicker('hideWidget');
+              console.log(new_url);
+              loadJSON(new_url);
+            })
+          });
+      function loadJSON(url) {
+        var xhttp;
+        xhttp=new XMLHttpRequest();
+        xhttp.onreadystatechange = function() {
+          if (this.readyState == 4 && this.status == 200) {
+            jsonText = xhttp.responseText
+            buildPage(jsonText, false);
+          } else {
+            $('#target').html('No builds available for selected date.');
+          }
+        };
+        xhttp.open("GET", url, true);
+        xhttp.send();
+      }
+      function buildPage(jsonText, is_rebuild) {
+        const corretto_domain = 'https://corretto.aws';
+        var data = JSON.parse(jsonText);
+        $('#datepicker').datepicker('update', data.date);
+
+        const select_version = document.getElementById('versionSelect');
+        const select_platform = document.getElementById('platformSelect');
+        const select_architecture = document.getElementById('architectureSelect');
+
+        const known_versions = new Set(['All Versions'])
+        const known_platforms = new Set(['All Platforms'])
+        const known_architectures = new Set(['All Architectures'])
+        $html = '';
```

Comment:
can we put this HTML stuff directly in HTML. And then in JavaScript, add a row for each artifact. (not blocking)

Reply:
Yeah, I think I had it that way in an early iteration and will change back and give it a try.